www/src/data/sidebars/doc-links.yaml

```diff
@@ -1,128 +1,221 @@
-- title: Quick Start
+- title: Quick start
   items:
-    - title: Getting Started
+    - title: Quick start
       link: /docs/
-    - title: Starters
-      link: /docs/gatsby-starters/
+    - title: Overview of the Docs*
+      link: /docs/overview/
+    - title: Main Tutorial (step-by-step)
+      link: /tutorial/
+- title: Deploying & Hosting
+  items:
+    - title: Preparing site for deployment*
+      link: /docs/preparing-for-deployment/
     - title: Deploying
       link: /docs/deploy-gatsby/
-    - title: Community
-      link: /docs/community/
-    - title: Awesome Gatsby
-      link: /docs/awesome-gatsby/
-- title: Core Concepts
-  items:
-    - title: Building with Components
-      link: /docs/building-with-components/
-    - title: Lifecycle APIs
-      link: /docs/gatsby-lifecycle-apis/
-    - title: Plugins
-      link: /docs/plugins/
-    - title: PRPL Pattern
-      link: /docs/prpl-pattern/
-    - title: Querying Data with GraphQL
-      link: /docs/querying-with-graphql/
+    - title: Path Prefix
+      link: /docs/path-prefix/
+    - title: How Gatsby Works with GitHub pages
+      link: /docs/how-gatsby-works-with-github-pages/
 - title: Guides
   items:
-    - title: Adding a 404 Page
-      link: /docs/add-404-page/
-    - title: Adding Images, Fonts, and Files
-      link: /docs/adding-images-fonts-files/
-    - title: Adding Images to a WordPress Site
-      link: /docs/image-tutorial/
-    - title: Babel.js
-      link: /docs/babel/
-    - title: Browser Support
-      link: /docs/browser-support/
-    - title: Building Apps with Gatsby
+    - title: Environment*
+      link: /docs/environment/
+      items:
+      - title: Basic hardware and software requirements*
+        link: /docs/basic-hardware-software-requirements/
+      - title: Browser Support
+        link: /docs/browser-support/
+      - title: Gatsby on Macs*
+        link: /docs/gatsby-on-macs/
+      - title: Gatsby on Windows
+        link: /docs/gatsby-on-windows/
+    - title: Customization*
+      link: /docs/customization/
+      items:
+        - title: Babel.js
+          link: /docs/babel/
+        - title: Custom webpack config
+          link: /docs/add-custom-webpack-config/
+        - title: Customizing html.js
+          link: /docs/custom-html/
+        - title: Environment variables
+          link: /docs/environment-variables/
+        - title: Structure of Gatsby default starter*
+          link: /docs/gatsby-default-starter-structure/
+    - title: Content*
+      link: /docs/content/
+      items:
+        - title: Adding a 404 Page
+          link: /docs/add-404-page/
+        - title: Adding Images, Fonts, and Files
```

Comment:
This is a confusing article in general -- in our analytics this has one of the highest bounce rates. It's sort of a bunch of different concepts mashed together. At some point, we should break this up into two / three different articles, eg:

* Using the `static` folder
* Importing files into Gatsby
    * Importing single files
    * Importing media content
    * Programmatically importing content using gatsby-source-filesystem.

Reply:
Agreed! Just updated phase 2 of the RFC to include this suggestion and others like it. Didn't think it made sense to replace the doc with a set of stub articles yet b/c we'll likely reuse the content. So I can create all the stub articles and then link back to the mother article and say "if you're creating this doc, refer to this original doc for relevant info" or something like that. Does that make sense?

Reply:
Ah, I see! And as far as gatsby-source-filesystem, would it make sense to say
a) "programmatically import content"
b) "programmatically import content via plugin
c) "`gatsby-source-filesystem` programmatic import"

Reply:
For this sort of thing, it'd be nice to have an introductory article that describes briefly the different ways of adding assets and the tradeoffs and then has detailed articles on each specific option.

This is the same pattern we want for many of these umbrella topics like deployment, headless CMSs, styling, etc.

Reply:
yes it is; the accordion level-1 headers don't have any page attached (@fk is going to work on making this possible), but the level-2 headers will have summary pages (already created stubs).